mvicore-diff/src/main/java/com/badoo/mvicore/ModelWatcher.kt

```diff
@@ -0,0 +1,52 @@
+package com.badoo.mvicore
+
+class ModelWatcher<T> private constructor(
+    private val watchers: List<Watcher<T, Any?>>
+) {
+    private var model: T? = null
+
+    operator fun invoke(value: T) {
```

Comment:
I suggest to rename `value` to `newModel`
```suggestion
    operator fun invoke(newModel: T) {
```